src/packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -44,7 +45,6 @@ export const predefinedGeneratorResolvers: PredefinedGeneratorResolvers = {
   'prisma-client-js': async (baseDir, version) => {
     let prismaClientDir = resolvePkg('@prisma/client', { cwd: baseDir })
     checkYarnVersion()
-    checkTypeScriptVersion()
```

Comment:
Why moving this to Generate CLI command?

This TS version should only apply for `prisma-client-js` I think?

For example it doesn't apply for `prisma-client-go`

Reply:
Good point, still sleeping